distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFtp.java

```diff
@@ -98,61 +112,165 @@ public void afterExam() throws Exception {
+     * Simple test verifying FTPS client with keystore can be connected and successfully complete
+     * SSL handshake with FTP server when clientAuth = "need"
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testFtpsWithKeystoreNeedClientAuth() throws Exception {
+        setClientAuthConfiguration(NEED);
+        FTPSClient client = createSecureClient(true);
+        assertTrue(client.sendNoOp());
+        disconnectClient(client);
     }
 
     /**
-     * Simple test verifying FTP client can be connected and configured successfully
-     * 
+     * Simple test verifying FTPS client without keystore can be connected and successfully complete
+     * SSL handshake with FTP server when clientAuth = "want"
+     *
      * @throws Exception
      */
     @Test
-    public void testFtp() throws Exception {
-        client = createInsecureClient();
+    public void testFtpsWithoutKeystoreWantClientAuth() throws Exception {
+        setClientAuthConfiguration(WANT);
+        FTPSClient client = createSecureClient(false);
         assertTrue(client.sendNoOp());
+        disconnectClient(client);
+    }
+
+    /**
+     * Simple test verifying FTPS client without keystore can be connected and successfully complete
+     * SSL handshake with FTP server when clientAuth = "need"
+     *
+     * @throws Exception
+     */
+    @Test(expected = SSLException.class)
+    public void testFtpsWithoutKeystoreNeedClientAuth() throws Exception {
+        setClientAuthConfiguration(NEED);
+        FTPSClient client = createSecureClient(false);
+        disconnectClient(client);
     }
 
     /**
      * Upload a file via insecure FTP for ingest using input stream
-     * 
+     *
      * @throws Exception
      */
     @Test
-    public void testInsecureFtpPut() throws Exception {
-        client = createInsecureClient();
+    public void testFtpPut() throws Exception {
+        setClientAuthConfiguration(WANT);
+        FTPClient client = createInsecureClient();
 
-        ftpPut(SAMPLE_DATA);
+        ftpPut(client, SAMPLE_DATA);
 
         // verify FTP PUT resulted in ingest, catalogued data
         Response response = executeOpenSearch("xml", "q=*", "count=100");
-        response.then().log().all().body("metacards.metacard.size()", equalTo(1));
+        response.then()
+                .body("metacards.metacard.size()", equalTo(1));
 
         // clean up test data
         TestCatalog.deleteMetacard(getMetacardIdFromResponse(response));
+        disconnectClient(client);
     }
 
     /**
      * Upload a file via insecure FTP for ingest using streaming method
-     * 
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testFtpPutStream() throws Exception {
+        setClientAuthConfiguration(WANT);
+        FTPClient client = createInsecureClient();
+
+        ftpPutStreaming(client,
+                IOUtils.toString(TestFtp.class.getClassLoader()
+                        .getResourceAsStream("/metacard1.xml")));
```

Comment:
This should be fine on Windows. Good job. Just adding a comment so I can refer to this later, just in case.